rdt-open-reader/src/main/java/io/ona/rdt/fragment/RDTJsonFormFragment.java

```diff
@@ -37,6 +38,8 @@
  */
 public class RDTJsonFormFragment extends JsonFormFragment implements RDTJsonFormFragmentContract.View {
 
+    private static final String STEP_8 = "step8";
```

Comment:
This is now unused. This file should ideally be unchanged.

Reply:
it has been removed.